addon/templates/components/nav-modal-binding.hbs

```diff
@@ -0,0 +1,6 @@
+{{#from-elsewhere name="modal" as |modal|}}
```

Comment:
I think you've flipping the binding and outlet concepts, the outlet should bind the passed modal into the liquid-if so that the animation occurs when the (now conditional) to-elsewhere sends the component

https://github.com/ciena-frost/ember-frost-modal/blob/master/addon/templates/components/frost-modal-outlet.hbs